docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md

```diff
@@ -54,25 +61,118 @@ This setting has precedence over each individual rule frequency. If a rule frequ
 
-## Create recording rules
-
-To create recording rules, follow these steps.
+### Steps
 
 1. Click **Alerts & IRM** -> **Alerting** ->
    **Alert rules**.
 1. Select **Rule type** -> **Recording**.
-1. Click **+New recording rule**.
+1. Scroll to the **Data-source managed section** and click **+New recording rule**.
+
+#### Enter recording rule name
+
+The recording rule name must be a Prometheus metric name and contain no whitespace.
+
+#### Define recording rule
+
+Select your data source and enter a query.
+
+#### Add namespace and group
+
+1. From the **Namespace** dropdown, select an existing rule namespace or add a new one.
+
+   Namespaces can contain one or more rule groups and only have an organizational purpose.
+
+1. From the **Group** dropdown, select an existing group within the selected namespace or add a new one.
 
-1. Enter recording rule name.
+   Newly created rules are appended to the end of the group. Rules within a group are run sequentially at a regular interval, with the same evaluation time.
 
-   The recording rule name must be a Prometheus metric name and contain no whitespace.
+#### Add labels
+
+1. Add custom labels selecting existing key-value pairs from the drop down, or add new labels by entering the new key or value.
 
-1. Define recording rule.
-   - Select your Loki or Prometheus data source.
-   - Enter a query.
-1. Add namespace and group.
-   - From the **Namespace** dropdown, select an existing rule namespace or add a new one. Namespaces can contain one or more rule groups and only have an organizational purpose.
-   - From the **Group** dropdown, select an existing group within the selected namespace or add a new one. Newly created rules are appended to the end of the group. Rules within a group are run sequentially at a regular interval, with the same evaluation time.
-1. Add labels.
-   - Add custom labels selecting existing key-value pairs from the drop down, or add new labels by entering the new key or value .
 1. Click **Save rule** to save the rule or **Save rule and exit** to save the rule and go back to the Alerting page.
+
+## Configure Grafana-managed recording rules
+
+To configure Grafana-managed recording rules, complete the following steps.
+
+### Before you begin
+
+If you are using Grafana OSS, enable the `grafanaManagedRecordingRules` feature flag.
+
+### Steps
+
+1. Click **Alerts & IRM** -> **Alerting** ->
+   **Alert rules**.
+1. Select **Rule type** -> **Recording**.
+1. Scroll to the **Grafana-managed section** and click **+New recording rule**.
+
+#### Enter a recording rule and metric name
+
+Enter a names to identify your recording rule and metric.
```

Comment:
we should add: The metric name must be a Prometheus metric name and contain no whitespace.